docs/processes/release/tasks/versions.md

```diff
@@ -0,0 +1,278 @@
+
+*   api.report.version: 2 digit release number. Note: If necessary, we may
+    include maintenance release number. (e.g "54", "481")
+*   api.report.prev.version: The previous version compared against the current
+    version. (e.g. "46")
+*   release.file.ver: This string is used for files names. For milestone
+    releases, use <2-digit major version> + "m" + <milestone number>, e.g.
+    "55m1". For release candidate, use <2-digit major version> + "rc", e.g.
+    54rc. For official releases, use full version numbers using under bar as the
+    separator, e.g. "54_1", "54_1_1".
+*   api.doc.version: The version displayed in API reference doc - use full
+    version number such as "60.1" for official releases, "60 Milestone 1" for
+    milestone 1.
+*   maven.pom.ver: The version used in ICU pom.xml files. Use full version
+    number such as "60.1" for official releases, "61.1-SNAPSHOT" until 61.1
+    release, after 60.1.
+
+3.
+[icu4j/main/classes/core/src/com/ibm/icu/util/VersionInfo.java](https://github.com/unicode-org/icu/blob/master/icu4j/main/classes/core/src/com/ibm/icu/util/VersionInfo.java)
+
+There is a static block starting at line 501 (as of 54.1) in the source file -
+
+/\*\*
+\* Initialize versions only after MAP_ has been created
+\*/
+static {
+UNICODE_1_0 = getInstance(1, 0, 0, 0);
+UNICODE_1_0_1 = getInstance(1, 0, 1, 0);
+UNICODE_1_1_0 = getInstance(1, 1, 0, 0);
+UNICODE_1_1_5 = getInstance(1, 1, 5, 0);
+UNICODE_2_0 = getInstance(2, 0, 0, 0);
+UNICODE_2_1_2 = getInstance(2, 1, 2, 0);
+UNICODE_2_1_5 = getInstance(2, 1, 5, 0);
+UNICODE_2_1_8 = getInstance(2, 1, 8, 0);
+UNICODE_2_1_9 = getInstance(2, 1, 9, 0);
+UNICODE_3_0 = getInstance(3, 0, 0, 0);
+UNICODE_3_0_1 = getInstance(3, 0, 1, 0);
+UNICODE_3_1_0 = getInstance(3, 1, 0, 0);
+UNICODE_3_1_1 = getInstance(3, 1, 1, 0);
+UNICODE_3_2 = getInstance(3, 2, 0, 0);
+UNICODE_4_0 = getInstance(4, 0, 0, 0);
+UNICODE_4_0_1 = getInstance(4, 0, 1, 0);
+UNICODE_4_1 = getInstance(4, 1, 0, 0);
+UNICODE_5_0 = getInstance(5, 0, 0, 0);
+UNICODE_5_1 = getInstance(5, 1, 0, 0);
+UNICODE_5_2 = getInstance(5, 2, 0, 0);
+UNICODE_6_0 = getInstance(6, 0, 0, 0);
+UNICODE_6_1 = getInstance(6, 1, 0, 0);
+UNICODE_6_2 = getInstance(6, 2, 0, 0);
+UNICODE_6_3 = getInstance(6, 3, 0, 0);
+UNICODE_7_0 = getInstance(7, 0, 0, 0);
+ICU_VERSION = getInstance(54, 1, 0, 0);
+ICU_DATA_VERSION = getInstance(54, 1, 0, 0);
+UNICODE_VERSION = UNICODE_7_0;
+UCOL_RUNTIME_VERSION = getInstance(8);
+UCOL_BUILDER_VERSION = getInstance(9);
+UCOL_TAILORINGS_VERSION = getInstance(1);
+}
+In the same file, starting at line 164 (as of 54.1) -
+/\*\*
+\* Data version string for ICU's internal data.
+\* Used for appending to data path (e.g. icudt43b)
+\* @internal
+\* @deprecated This API is ICU internal only.
+\*/
+@Deprecated
+public static final String ICU_DATA_VERSION_PATH = "54b";
+4. [icu4j/pom.xml](http://source.icu-project.org/repos/icu/trunk/icu4j/pom.xml)
+(before ICU4J 60 only)
+<groupId>com.ibm.icu</groupId>
```

Comment:
code block

Reply:
Done - using <pre> in this file, because some blocks need <b> (used <b> on Sites).